Apps/Src/BlinkLights.c

```diff
@@ -0,0 +1,93 @@
+#include "Tasks.h"
+#include "Lights.h"
+
+void Task_BlinkLight(void* p_arg){
```

Comment:
What is the expected behavior of this task? It would be nice if there was a comment